operator/controllers/csiscaleoperator_controller.go

```diff
@@ -764,9 +766,10 @@ func (r *CSIScaleOperatorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	//Do not restart driver pods when the configmap contains invalid Envs.
 	shouldRequeueOnCreateOrDelete := func(cfgmapData map[string]string) bool {
 		for key := range cfgmapData {
-			if strings.HasPrefix(strings.ToUpper(key), config.EnvVarPrefix) || 
+			if strings.HasPrefix(strings.ToUpper(key), config.EnvVarPrefix) ||
 				strings.ToUpper(key) == config.DaemonSetUpgradeMaxUnavailableKey ||
-				strings.ToUpper(key) == config.HostNetworkKey{
+				strings.ToUpper(key) == config.HostNetworkKey ||
+				strings.ToUpper(key) == config.DriverCPULimits {
```

Comment:
We need to check for other keys also here. Similar change is required in 3 other places below.